src/redux/reducers/cellSets/cellSetsClusteringUpdated.js

```diff
@@ -0,0 +1,15 @@
+import { createHierarchyFromTree, createPropertiesFromTree } from './helpers';
+
+const cellSetsClusteringUpdated = (state, action) => {
+  const { data } = action.payload;
+
+  return {
+    ...state,
+    loading: false,
+    updatingClustering: false,
+    hierarchy: createHierarchyFromTree(data),
```

Comment:
have you tested in here that if the user reclusters, the Scratchpad cluster entry and the clusters under it, if they are any, do not disappear?

Reply:
You're right, I'll add that.